### ISSUE TITLE
Article focus

### ISSUE THREAD

**Issue by chrisdemars** [a11yproject/a11yproject.com#516]
I give a talk at conferences on focus and it was published on David Walsh's blog. I would love to have it published as an article for the A11y Project, if no one objects.

**Comment by svinkle** [comment]
@chrisdemars Of course! Could you provide a link to your article for us to check out before-hand? Thanks!

**Comment by chrisdemars** [comment]
No problem @svinkle, want me to post here or send you an email?

**Comment by svinkle** [comment]
@chrisdemars Right here's good, if you're cool with that. 👍

**Comment by chrisdemars** [comment]
No problem @svinkle, [Focusing on Focus](https://davidwalsh.name/css-focus)

**Comment by svinkle** [comment]
@chrisdemars Great article! `:focus-ring` is a hot topic right now so this would be a fantastic add to the site here.

A few things I noticed that should be addressed before submitting:

- First sentence has "a11y" -- please read this recent discussion: #509 (tl;dr: write out "accessibility" instead of using the numeronym.)
- Change
  > There needs to be a way for accessible and non-accessible users to be able to see where they are during interactions.

  To
  > Everyone, including sighted users, people with low-vision, or those who rely on the keyboard, needs to understand where they are during interactions.

- Change
  >  It would suck if you went to start typing your password but had no visual indication of where you were actually typing your password.

  To
  >  It would be a major irritation when typing your password to realize you were typing in the wrong place from a lack of visual indiction.

- This line feels like there's something missing, "but is has to have"?
  > Now that being said, it doesn’t say a blue ring, this is afforded by the browser, but is has to have some type of visual distinction.

- Change
  > I touched briefly on :focus-ring earlier, it is currently in the CSS 4 spec, it landed on the 18th of May.

  To
  > I touched briefly on :focus-ring earlier. It's currently in the CSS 4 spec, added on the 18th of May.

- Could you add a link to the `:focus-ring` polyfill somewhere around this area where it's first mentioned?
- Remove
  > Even others who use some type of assistive technology if I had to bet on it.

- Change
  > Some mouse users, on the other hand, don’t like having a focus-ring around custom elements, like custom buttons.

  To
  > Some mouse users don’t like having a focus-ring around custom elements, such as custom buttons.

As a bonus for our audience, would you be willing to create a CodePen demo (or some other code demo site) for people to see this in action? We'd link out to this from the Patterns section.

Thanks, Chris!

**Comment by gio66613** [comment]
[![Stories in Ready](https://badge.waffle.io/gio66613/newsecularorder.png?label=ready&title=Ready)](http://waffle.io/gio66613/newsecularorder)
readme.rst

.. image:: https://badge.waffle.io/gio66613/newsecularorder.png?label=ready&title=Ready 
 :target: https://waffle.io/gio66613/newsecularorder 
 :alt: 'Stories in Ready'
readme.rdoc

{<img alt='Stories in Ready' src='https://badge.waffle.io/gio66613/newsecularorder.png?label=ready&title=Ready' />}[https://waffle.io/gio66613/newsecularorder]

**Comment by svinkle** [comment]
@chrisdemars Any update on this? Would like to publish. 🙂

**Comment by chrisdemars** [comment]
Hey @svinkle, I am def going to work on it. Just have been crazy busy these past few months. I am going to have to make a separate one as to not rewrite the original.

**Comment by stefanjudis** [comment]
Hey hey. 👋🏻

Just came across and I'm also digging `:focus-ring` just wanted to mention that there was a spec change and renaming to `:focus-indicator`. that's probably worth mentioning. :)
 
👉🏻 https://github.com/w3c/csswg-drafts/issues/2036

**Comment by joe-watkins** [comment]
Good find @stefanjudis I wasn't aware they had changed the name. Wonder what this means for `-moz-focusring`.

**Comment by chrisdemars** [comment]
I saw Marcy talking about that couple weeks ago so I will be sure to change
it. I also have to change that in my conference talk as well.
